spec/lib/facilities/ppms/responses/response_spec.rb

```diff
@@ -0,0 +1,259 @@
+      'HighPerformance' => false,
+      'Fqhc' => false,
+      'AccountableCareOrganization' => false,
+      'IsAcceptingNewPatients' => false,
+      'ProviderEthnicity' => 'Unknown',
+      'ProviderGender' => 'Male',
+      'Religion' => nil,
+      'OrganizationId' => nil,
+      'ServiceProviderType' => 'None',
+      'SpecialInstruction' => nil,
+      'OwnedCareSiteName' => nil,
+      'OrganizationFax' => '(540) 665-0411',
+      'OrganizationStatus' => 'Active',
+      'IsExternal' => false,
+      'InternalType' => '0',
+      'LicensingJuristicion' => nil,
+      'CanCreateHealthCareOrders' => false,
+      'InternalAppointmentStatus' => 'FullTime',
+      'ExternalHealthProviderType' => 'CommunityHospital',
+      'OnLeie' => false,
+      'ExternalInstitutionDeaNumber' => nil,
+      'ExternalLeieCheckDate' => nil,
+      'ValidationSource' => nil,
+      'ContactMethod' => 'Any',
+      'BulkEmails' => true,
+      'BulkMails' => false,
+      'Emails' => true,
+      'Mails' => true,
+      'PhoneCalls' => true,
+      'Faxes' => true,
+      'ModifiedOnDate' => '2018-08-02T04:27:10Z',
+      'TerminationDate' => '0001-01-01T00:00:00Z',
+      'ProviderSpecialties' => []
+    }
+  end
+
+  let(:response) { Facilities::PPMS::Response.new(response_body, 200) }
+  let(:new_provider) { response.new_provider }
+  let(:provider_carsites_response) { Facilities::PPMS::Response.new(caresites_response_body, 200) }
+  let(:provider_services_response) { Facilities::PPMS::Response.new(services_response_body, 200) }
+  let(:provider_specialties_response) { Facilities::PPMS::Response.new(specialties_response_body, 200) }
+  let(:bbox) { { bbox: [-79, 38, -77, 39] } }
+
+  let(:faraday_response) { instance_double('Faraday::Response') }
+  let(:provider_locator_response) { Facilities::PPMS::Response.from_provider_locator(faraday_response, bbox) }
+
+  describe 'getting data' do
+    context 'with a successful response' do
+      it 'has the proper response object attributes' do
+        expect(response).not_to be(nil)
+        expect(response.body).not_to be(nil)
+        expect(response.get_body).not_to be(nil)
+        expect(response.status).to eq(200)
+      end
+
+      it 'has the proper caresites identifiers' do
+        expect(provider_carsites_response).not_to be(nil)
+        expect(provider_carsites_response.body).not_to be(nil)
+        expect(provider_carsites_response.status).to eq(200)
+      end
+
+      it 'has the proper provider services identifiers' do
+        expect(provider_services_response).not_to be(nil)
+        expect(provider_services_response.body).not_to be(nil)
+        expect(provider_services_response.status).to eq(200)
+      end
+
+      it 'has the proper provider specialties identifiers' do
+        expect(provider_specialties_response).not_to be(nil)
+        expect(provider_specialties_response.body).not_to be(nil)
+        expect(provider_specialties_response.status).to eq(200)
+      end
```

Comment:
These are all just testing that the initialize.  That is, the `body` and `status` are just set as the arguments provided to `new`. 

I'd say we just need one of these that looks something like this:
```
    it 'has the proper response object attributes' do
        expect(response).not_to be(nil)
        expect(response.body).to eq(response_body)
        expect(response.get_body).to eq(response_body)
        expect(response.status).to eq(200)
      end
```
I'd also like to see specs that test the other methods: `self.from_provider_locator` and `self.map_provider_list`.

I see above you've declared `let(:provider_locator_response)`, but `provider_locator_response` isn't used anywhere.

Reply:
Tests and file paths updated ✔️